lib/pure/strutils.nim

```diff
@@ -2007,6 +2007,16 @@ proc formatFloat*(f: float, format: FloatFormatMode = ffDefault,
   ## after the decimal point for Nim's ``float`` type.
   ##
   ## If ``precision == 0``, it tries to format it nicely.
+  ##
+  ## Examples:
+  ##
+  ## .. code-block:: nim
+  ##
+  ##    let x = 123.456
+  ##    echo x.formatFloat()                # 123.4560000000000
```

Comment:
Can we please change this to a ``doAssert`` call?

@Araq can you ask people to use doAssert for such examples from now on? That way we can test the examples in the future easily.

Reply:
Fine with me.